ConwaysGameOfLife.Tests/RulesShould.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using Moq;
+using Xunit;
+
+namespace ConwaysGameOfLife.Tests
+{
+    public class RulesShould
+    {
+        [Fact]
+        public void WhenAnyLiveCellHasFewerThan2LiveNeighbours_ItDies()
+        {
+            var universe = new Universe(3, 3);
+            var rules = new Rules(universe);
+            var cell = new Cell(It.IsAny<CellState>(), 1, 1);

```

Comment:
Do we need a `Cell` or just a `Location` here? What is the purpose of stubbing the `CellState`?